imports/plugins/included/search-mongo/server/methods/searchcollections.js

```diff
@@ -183,9 +183,9 @@ export function buildOrderSearchRecord(orderId) {
   ) || {};
 
   orderSearch.billingName = shopBilling.address && shopBilling.address.fullName;
-  orderSearch.billingPhone = shopBilling.address && shopBilling.address.phone.replace(/\D/g, "");
```

Comment:
This change is going to break phone number searches. We store them as just digits so that (323) will match with 323 and 323-.

Reply:
Okay understood